tests/cli/test_cli_core.py

```diff
@@ -146,11 +150,21 @@ def test_cancel_not_running(
     trailblazer_context: Dict[str, MockStore],
     caplog,
     failed_analysis_case_name: str,
+    mocker,
     process_exit_success: int,
+    slurm_squeue_output: Dict[str, Path],
 ):
     """Test cancelling an analysis, which is not running."""
     caplog.set_level("ERROR")
 
+    # GIVEN SLURM squeue output for an analysis
+    mocker.patch(
+        FUNC_GET_SLURM_SQUEUE_OUTPUT_PATH,
+        return_value=subprocess.check_output(
+            ["cat", slurm_squeue_output.get(failed_analysis_case_name)]
+        ).decode("utf-8"),
+    )
```

Comment:
Neat ⭐

Reply:
👍 . Was important to move it from the Store